helm/templates/server/log-configmap.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.server.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "pinot.server.fullname" . }}-log-config
+  labels:
+    app: {{ include "pinot.name" . }}
+    chart: {{ include "pinot.chart" . }}
+    component: {{ .Values.server.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  pinot-server-log4j2.xml: |-
+    <Configuration>
+      <Appenders>
+        <Console name="console" target="SYSTEM_OUT">
+          <PatternLayout>
+            <pattern>%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c{1}] [%t] %m%n</pattern>
+          </PatternLayout>
+        </Console>
+      </Appenders>
+      <Loggers>
+        <Root level="info" additivity="false">
```

Comment:
Can we parameterize the log level via values?

Reply:
done.